src/main/java/com/google/sps/utility/FreeTimeUtility.java

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utility;
+
+import com.google.sps.data.CalendarDataResponse;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+/** Class containing the user's free hours. */
+public final class FreeTimeUtility {
+
+  private final Date startDate;
+  private List<Long> workHoursPerDay;
+  private List<Long> personalHoursPerDay;
```

Comment:
These lists can just be local variables in `getCalendarDataResponse`. You are not updating them in `addEvent`, and you are not using them for any other reason except to get the response.

Reply:
`Resolved`